packages/cli/src/models/status/StatusChecker.js

```diff
@@ -113,9 +118,9 @@ export default class StatusChecker {
     if (address !== expected) this.visitor.onMismatchingContractAddress(expected, address, { alias, address })
   }
 
-  _checkImplementationBytecode(alias, address) {
+  async _checkImplementationBytecode(alias, address) {
     const expected = this.networkFile.contract(alias).bodyBytecodeHash
-    const observed = bytecodeDigest(web3.eth.getCode(address))
+    const observed = bytecodeDigest(await promisify(web3.eth.getCode.bind(web3.eth))(address))
```

Comment:
nice catch! we should have a common file providing this promisified functions of web3

Reply:
Agree. We should also consider using [this](http://bluebirdjs.com/docs/api/promise.promisifyall.html)